pkg/broker/broker.go

```diff
@@ -39,6 +41,25 @@ func (b *InMemoryBroker) Send(room string, message messaging.Message) {
 	}
 }
 
+func (b *InMemoryBroker) SendControl(room string, payload interface{}) {
+	b.mutex.RLock()
+	defer b.mutex.RUnlock()
+
+	payloadJson, err := json.Marshal(payload)
+	if err != nil {
+		b.logger.Error("failed to marshal control payload", logging.Fields{"room": room})
+	}
+
+	b.broadcast(
+		messaging.Message{
+			From:    "tarpon",
```

Comment:
This should be moved to some constants and anyway not broker's responsibility to set it.